### PR TITLE
RTD documentation

### PR DESCRIPTION
**Pull request by rjuntima**
Sariya and MK's RTD documentation

### CODE REVIEW

**Comment by cissieAB** [comment]
I accept this PR as squash in case all 221 commits flushed in.

File: lumache.py
Comment (wdconinc) [inline lumache.py:1]: Can you remove this again?

File: pyproject.toml
Comment (wdconinc) [inline pyproject.toml:1]: Can you remove this again?

**Review by wdconinc — COMMENTED** [review]
Two unrelated files were included in this PR.

**Comment by cissieAB** [comment]
> Two unrelated files were included in this PR.

Address in https://github.com/JeffersonLab/JANA2/commit/27be06f2e976040c5db4cd7d19ca9bba3956d735. Thanks for reminding!

**Pull request merged**